examples/office/importers/utrade/tests/expected/d60c05338c9bb13c44249d0c9adefdde4ea273a5.beancount

```diff
@@ -1,24 +1,25 @@
+;;; document filing path: Assets/US/UTrade/2015-02-25-utrade.UTrade20150225.csv
```

Comment:
What about file_date()?
That one's an important piece of data that is produced from the extractors, because it helps determine the output filename and date of the document / application.

Reply:
I opted for combining file_account, file_date, and file_date into "document filing path" (I arbitrarily choose this wording, if you have a better one I can change it) as the ingest framework does, it seems natural.

Reply:
I'd rather see them as separate fields. Also, think about the possibility of the file naming scheme changing in the future to not have the date on it.

Reply:
Sure, I'll change this.